daffodil-io/src/main/scala/org/apache/daffodil/processors/charset/DaffodilCharsetProvider.scala

```diff
@@ -25,7 +25,14 @@ object DaffodilCharsetProvider {
     BitsCharset5BitPackedLSBF,
     BitsCharset6BitDFI264DUI001,
     BitsCharset6BitDFI311DUI002,
+    BitsCharset3BitDFI336DUI001,
+    BitsCharset4BitDFI746DUI002,
+    BitsCharset3BitDFI746DUI002,
+    BitsCharset3BitDFI747DUI001,
+    BitsCharset5BitDFI769DUI002,
```

Comment:
Might be worth sorting these just for extra clarity.

Reply:
They are sorted by the DFI number. Not the length. But given the mixture of different encodings here, some of which are these obscure DFI/DUI things, some of which are more understandable things, I take your point. i'll sort these alphanumerically.